tools/antithesis/test_format.dockerfile

```diff
@@ -5,7 +5,7 @@ RUN mkdir -p bin/test/format
 RUN mkdir -p /data/RUNDIR
 COPY cmake_build/test/format/t bin/test/format/
 COPY tools/voidstar/lib/libvoidstar.so tools/voidstar/lib/
-COPY cmake_build/libwiredtiger.so.11.2.0 bin/
+COPY cmake_build/libwiredtiger.so.* bin/
```

Comment:
See 69a5506344e6783b081d043e62e2e4d5fd18a39b, I went for a stricter match when copying.